### PR TITLE
Randomize cdi namespace in tests.

### PR DESCRIPTION
**Pull request by awels**
Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:
This PR randomizes the namespace cdi is installed in. This ensures our tests are compatible with different namespaces.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/utils/configmaps.go [hunk 1]
```diff
@@ -12,20 +12,20 @@ import (
 	"kubevirt.io/containerized-data-importer/pkg/util"
 )
 
-const registryHostname = RegistryHostName + "." + RegistryHostNs
+//const registryHostname = RegistryHostName + "." + RegistryHostNs
```
Comment (mhenriks) [inline tests/utils/configmaps.go]: Should this be deleted?

Comment (awels) [reply tests/utils/configmaps.go]: Yes

**Pull request merged**